server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/DSNBounce.java

```diff
@@ -220,8 +220,7 @@ public String getMessage(Mail originalMail) throws MessagingException {
 
     @Override
     public Optional<MailAddress> getSender() throws MessagingException {
-        return SpecialAddressesUtils.from(this)
-                .getFirstSpecialAddressIfMatchingOrGivenAddress(getInitParameters().getSender(), RedirectNotify.SENDER_ALLOWED_SPECIALS);
+        return SpecialAddressesUtils.from(this).getFirstSpecialAddressIfMatchingOrGivenAddress(Optional.of(getInitParameters().getSender().orElse("postmaster")), RedirectNotify.SENDER_ALLOWED_SPECIALS);
```

Comment:
this line is too long

Reply:
Please comment in the original pull request in the description. Not my work here.